queue/queue.py

```diff
@@ -3,13 +3,23 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = []
 
   def enqueue(self, item):
-    pass
+    return self.storage.insert(0, item)
   
   def dequeue(self):
-    pass
+    if len(self.storage):
+      return self.storage.pop()
```

Comment:
To elaborate on the above critique, here is an example:

When we remove the item from the front of the list/array, we force all other values inside that list/array to be re-index'd (a linear time operation).  If we were using a LinkedList() instead, this would be a constant time operation and run much faster.  This is a strong reason why we use LinkedLists as the storage structure for implementing a Queue or Stack.